lib/Imagine/Filter/Transformation.php

```diff
@@ -96,7 +96,7 @@ public function getFilters()
     {
         if (null === $this->sorted) {
             ksort($this->filters);
-            $this->sorted = call_user_func_array('array_merge', $this->filters);
+            $this->sorted = array_merge($this->filters);
```

Comment:
This patch is wrong, see the result of the code above here http://3v4l.org/jeprO

```
$array = [['a'], ['b'], ['c']];
var_dump(call_user_func_array('array_merge', $array));
var_dump(array_merge($array));
```